Index: src/vendor/magento/module-ui/view/base/web/js/lib/view/utils/dom-observer.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/view/base/web/js/lib/view/utils/dom-observer.js b/view/base/web/js/lib/view/utils/dom-observer.js
--- a/view/base/web/js/lib/view/utils/dom-observer.js	
+++ b/view/base/web/js/lib/view/utils/dom-observer.js	(date 1713248057631)
@@ -149,7 +149,7 @@
         _.each(watchers.selectors, function (listeners, selector) {
             for (let data of listeners) {
                 if (!data.ctx.contains(node) || !$(node, data.ctx).is(selector)) {
-                    break;
+                    continue;
                 }
 
                 if (data.type === 'add') {
